Assets/Scripts/Audio/AudioLoader.cs

```diff
@@ -6,38 +6,40 @@
 using UnityEngine;
```

Comment:
In the header, you don't need to leave a field for dependencies if there are none.